botfront/imports/ui/components/stories/StoryGroupTree.jsx

```diff
@@ -21,12 +22,19 @@ export default function StoryGroupTree(props) {
         stories,
         isStoryDeletable,
     } = props;
+
     const [deletionModalVisible, setDeletionModalVisible] = useState(false);
     const [mouseDown, setMouseDown] = useState(false);
+
+    const [selectedNewItem, setSelectedNewItem] = useState(false);
```

Comment:
Adding two state variables and two useEffects to implement scrollTo is overkill IMO. You're making the "don't scroll" behavior an exception, while it should be the default; instead the scroll behavior should be implemented from the location it originates, i.e. SearchBar.jsx.

Getting the DOM element and scrolling to it can be done from _anywhere_, including as a callback for the activeStories mutation. Needing to expand groups is no excuse either, the updateGroup method is exposed in the ConversationOptionsContext context.